lib/formfields/admin/domains/formfield.domains_add.php

```diff
@@ -174,7 +174,7 @@
 						'value' => array()
 					),
 					'http2' => array(
-						'visible' => ($ssl_ipsandports != '' ? true : false) && Settings::Get('system.webserver') != 'lighttpd',
+						'visible' => ($ssl_ipsandports != '' ? true : false) && Settings::Get('system.webserver') != 'lighttpd' && Settings::Get('system.http2_support') == '1',
```

Comment:
no need to check for webserver != 'lighttpd' imho because the setting is not available for lighttpd and thus cannot be activated anyway. Also, in case lighttp includes http2 support this would be a hard-to-find-thing. Just a check for http2-support should be enough